packages/framework/framework/tsconfig.json

```diff
@@ -1,23 +1,8 @@
 {
+  "extends": "@fluentui-react-native/scripts/tsconfig.json",
   "compilerOptions": {
-    "baseUrl": ".",
-    "target": "es5",
```

Comment:
target values are different, not sure if that's concerning to you. seems like aligning es versions makes sense though